Userland/Services/WebDriver/Client.cpp

```diff
@@ -641,6 +669,9 @@ Web::WebDriver::Response Client::delete_all_cookies(Web::WebDriver::Parameters p
     return session->web_content_connection().delete_all_cookies();
 }
 
+// 15.7 Perform Actions, https://w3c.github.io/webdriver/#dfn-perform-actions
+// POST	/session/{session id}/actions
```

Comment:
The stub method for this seems to be missing?

Reply:
#1261 has the stub for this method

Reply:
I can omit the Perform Actions